api/v1alpha1/common_types.go

```diff
@@ -55,7 +55,7 @@ type SelectorSpec struct {
 	// +optional
 	AnnotationSelectors map[string]string `json:"annotationSelectors,omitempty"`
 
-	// PodPhaseSelector is a set of condition of a pod at the current time.
+	// PhaseSelector is a set of condition of a pod at the current time.
 	// supported value: Pending / Running / Succeeded / Failed / Unknown
 	PodPhaseSelectors []string `json:"phaseSelector,omitempty"`
```

Comment:
```suggestion
	PodPhaseSelectors []string `json:"phaseSelectors,omitempty"`
```

Reply:
I think this change would break the backward compatible.

Reply:
I think this change would break the backward compatible.

Reply:
I guess this field hasn't been used now.

Reply:
It's okay to me if this hasn' been used.